src/sonic-config-engine/minigraph.py

```diff
@@ -1346,6 +1349,11 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     for port in ports.values():
         port['pfc_asym'] = 'off'
 
+    # make admin status to down if port not parsed from minigraph
+    for port in ports.values():
+        if port['alias'] not in port_alias_list:
```

Comment:
Does this impact multi-asic platforms where there are backplane ports? @arlakshm ?